installer/templates/phx_web/templates/layout/app.html.eex

```diff
@@ -4,9 +4,9 @@
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <%%= csrf_meta_tag() %>
     <title><%= app_module %> · Phoenix Framework</title>
     <link rel="stylesheet" href="<%%= Routes.static_path(@conn, "/css/app.css") %>"/>
-    <%%= csrf_meta_tag() %>
```

Comment:
Naive Q - Possible this was put here for a specific reason?

Reply:
This was added to `app.html.eex` before the `phx.new --live` feature was added. I think this should now be remove altogether from `app.html.eex` since `phx.new --live` apps use `root.html.leex`.

Reply:
This change isn't in the `--live` app template, but in the regular `phx_web` template - simply moving the meta tag next to the others. Can't think of any specific reason why it was put at the end of `<head>`...

Reply:
Ah I found what I was thinking about...So your change looks fine!

https://github.com/phoenixframework/phoenix_live_view/pull/577/files